ethereum_history_api/circuits/lib/src/receipt.nr

```diff
@@ -24,9 +24,9 @@ struct TxReceipt<LOG_NUM, MAX_LOG_DATA_LEN> {
     logs_bloom: [u8; BLOOM_FILTER_LEN]
 }
 
-struct TxReceiptProof<MAX_RECEIPT_PROOF_LEN, MAX_RECEIPT_RLP_LEN> {
+struct TxReceiptProof {
```

Comment:
Why removing generic arguments?
We still might need very log values, right?

Reply:
@LogvinovLeon

Reply:
The library for now does not have support for configuring those values so we decided to not pass them here also. The lib was built with an assumption that the length of branch node is bigger than the length of a value node.
Which is true for account & storage, but now for receipts or transactions. This is something we want to support in the future, but most of the receipts do fit now. We also have an issue for it